README.md

```diff
@@ -400,6 +400,38 @@ How to choose between all these features? Here are some considerations:
     }
 
 
+    ```
+    </details>
+* `Time` maps to/from i32 or a newtype around it and represents the time‐of‐day value on its own, without any date or regional offset context. Attempting to apply or change a time zone on Time columns has no effect and is not supported.
```

Comment:
It's not a time-of-day, as it is modelled after MySQL's time, which is kinda "duration" or "interval" type: https://dev.mysql.com/doc/refman/8.4/en/time.html. See also: https://clickhouse.com/docs/sql-reference/data-types/time; we can just copy:

> The Time data type is used to store a time value independent of any calendar date. It is ideal for representing daily schedules, event times, or any situation where only the time component (hours, minutes, seconds) is important.